### PR TITLE
edits + additions to networking docs

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: 1.8/administration/overlay-networks/index.md [hunk 1]
```diff
@@ -38,6 +33,41 @@ The components of the overlay network interact in the following ways:
 
 - The CNI isolator is used for the Mesos containerizer. [DNI](https://docs.docker.com/engine/userguide/networking/dockernetworks/) is used for the Docker containerizer, shelling out to the Docker daemon.
 
-- For intra-node IP discovery we use an overlay orchestrator called navstar. This operator-facing system component is responsible for programming the overlay backend using a library called [lashup](https://github.com/dcos/lashup) that implements a gossip protocol to disseminate and coordinate overlay routing information among all Mesos agents in the DC/OS cluster.
+- An overlay orchestrator ([Navstar](https://github.com/dcos/navstar)) performs intra-node IP discovery. This operator-facing system component uses a library, ([lashup](https://github.com/dcos/lashup)) to program the overlay backend. Lashup implements a gossip protocol to disseminate and coordinate overlay routing information among all Mesos agents in the DC/OS cluster.
+
+**Note:** Your network must adhere to the [DC/OS system requirements](https://dcos.io/docs/1.8/administration/installing/custom/system-requirements/) to use DC/OS overlay networks.
+
+# Navstar DNS
```
Comment (sascala) [inline 1.8/administration/overlay-networks/index.md]: Why is this information useful to an end-user? How can I pull up these entries?

File: 1.8/administration/overlay-networks/index.md [hunk 2]
```diff
@@ -38,6 +33,41 @@ The components of the overlay network interact in the following ways:
...
+
+  Keep this limitation in mind when debugging issues on frameworks that use an overlay network and you see the `TASK_FAILED` message.
+
+* The DC/OS overlay network uses Linux bridge devices on agents to connect Mesos and Docker containers to the overlay network. The names of these bridge devices are derived from the overlay network name. Since Linux has a limitation of fifteen characters on network device names, there is a character limit of thirteen characters for the overlay network name (two characters are used to distinguish between a CNI bridge and a Docker bridge on the overlay network).
+
+* Certain names are reserved and cannot be used as DC/OS overlay network names, primarily because DC/OS overlay uses Docker networking underneath to connect Docker containers to the overlay, which in turn reserves certain network names. The reserved names are: `host`, `bridge` and `default`.
+
+* Marathon health checks will not work with certain overlay network configurations. If you are not using the default DC/OS overlay network configuration and Marathon does not have access to the overlay network, health checks will fail consistently even if the service is healthy.
```
Comment (sascala) [inline 1.8/administration/overlay-networks/index.md]: How can I allow Marathon to access the overlay network?

File: 1.8/administration/overlay-networks/ip-per-container.md [hunk 3]
```diff
@@ -66,11 +64,11 @@ You can override the default network or add additional overlay networks by modif
           prefix: 24
 ```
```
Comment (sascala) [inline 1.8/administration/overlay-networks/ip-per-container.md]: Can a user do this with a cloud installation of DC/OS?

**Comment by sascala** [comment]
@sargun Could you please take a look over the sections on navstar and limitations on `1.8/administration/overlay-networks/index.md`?

File: 1.8/administration/overlay-networks/ip-per-container.md (hunk 3 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/overlay-networks/ip-per-container.md]: yes, if he installs manually. This is configurable in the config.yaml file. It is not accessible if you deploy with a standard cloud template (in which case the default "dcos" network is included", but it is accessible if you just deploy manually configuring the file (locally on in the cloud)

Comment (sascala) [inline 1.8/administration/overlay-networks/ip-per-container.md]: Thank you, @fernandosanchezmunoz

**Comment by sascala** [comment]
@joel-hamill These pages still need some polishing, but I'd like to publish these limitations and other edits in time for 1.8.

File: 1.8/administration/overlay-networks/ip-per-container.md [hunk 4]
```diff
@@ -16,18 +16,16 @@ The overlay network feature is enabled by default in DC/OS. The default configur
...
 
-These values can be configured to adapt to each installation’s needs. You can also add more overlay networks in addition to the default one, or modify/delete the existing overlay network. Currently, the addition/deletion of an overlay network is supported only at the time of installation. The next section describes how you can add more overlay networks to the existing default configuration.
+These values can be configured to adapt to each installation’s needs. You can also add more overlay networks in addition to the default one, or modify/delete the existing overlay network. Currently, the addition/deletion of an overlay network is supported only at install time. The next section describes how you can add more overlay networks to the existing default configuration.
```
Comment (joel-hamill) [inline 1.8/administration/overlay-networks/ip-per-container.md]: `You can modify the default overlay network configuration and add more overlay networks to fit your needs.`

**Comment by joel-hamill** [comment]
Made 1 comment, otherwise looks good to me!

**Comment by sascala** [comment]
Awesome! I made the change and can merge or you can.

**Pull request merged**